scenarios/19c_The_Oracle.cfg

```diff
@@ -1305,9 +1489,49 @@
                 description=_"Death of Lady Lorin"
                 condition=lose
             [/objective]
+            {TURNS_RUN_OUT}
+            {OBJECTIVE_NOTES _"The scenario will continue even if Matthias Ramon dies; it is the keep upon which he stands that requires greater protection."}
         [/objectives]
     [/event]
 
+    [event]
+        name=attack
+        [filter_second]
+            id=Matthias Ramon
+        [/filter_second]
+        {MESSAGE (Matthias Ramon) () (Matthias Ramon) _"I am under attack! To me, men! We must protect the Oracle together!"}
+        [message]
+            side=2
+            # Make sure he is not talking to himself:
+            [not]
+                id=Matthias Ramon
+            [/not]
+            message=_"On our way, sir!"
+        [/message]
+        [gold]
+            side=2
+            amount={ON_DIFFICULTY 60 40 20}
+        [/gold]
+        [modify_side]
+            side=2
+            # Make sure to keep this higher than their starting income:
+            {INCOME 9 8 7}
+        [/modify_side]
+        [move_unit]
+            side=2
+            to_x=19
+            to_y=23
+        [/move_unit]
+    [/event]
+
+    [event]
+        name=last breath
+        [filter]
+            id=Matthias Ramon
+        [/filter]
+        {MESSAGE (Matthias Ramon) () (Matthias Ramon) _"The attackers have slain me! Hurry, to my keep, men! You must protect the Oracle for me in my stead!"}
```

Comment:
Kind of funny that he talks after being slain (according to his dialogue), but it's fine. I think it's ironic anyway.

Reply:
oh yeah I noticed that as well on a re-read but figured it was just kind of an issue with all "last breath" events in general... :-P